src/SDKs/Network/Network.Tests/Tests/LoadBalancerTests.cs

```diff
@@ -1435,141 +1435,5 @@ public void LoadBalancerNatPoolTest()
                 networkManagementClient.PublicIPAddresses.Delete(resourceGroupName, lbPublicIpName);
              }
         }
-
-        [Fact(Skip="Disable tests")]
```

Comment:
Why are these being deleted? Was this service deprecated?

Reply:
OutboundNatRules were never made public
https://github.com/Azure/azure-rest-api-specs/pull/3592

Reply:
Specific comment: https://github.com/Azure/azure-rest-api-specs/pull/3592#discussion_r210762102